datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -164,4 +183,18 @@ private FlowIdentifier notify(String selector, SdxEvent acceptable) {
         }
 
     }
+
+    private String getBackupLocation(String datalakeName) {
+        String backupLocation = null;
+        DetailedEnvironmentResponse detailedEnvironmentResponse = environmentClientService.getByName(datalakeName);
+        if (!Strings.isNullOrEmpty(detailedEnvironmentResponse.getBackup().getStorageLocation())) {
+            backupLocation = detailedEnvironmentResponse.getBackup().getStorageLocation();
+        } else {
+            backupLocation = detailedEnvironmentResponse.getTelemetry().getLogging().getStorageLocation();
```

Comment:
telemetry can be null, for example on YCloud, but also many people test without telemetry info

Reply:
will address